src/pages/sign-up.page.js

```diff
@@ -140,6 +140,13 @@ export default class SignUpPage extends Component {
       });
   }
 
+  // ALLCLEAR-274
+  // Eventually add functionality to parse the error message and display
+  // properly. For now hide the error messages from the user
+  parseError() {
+    // return this.state.error === true ? <p className="error">{this.state.message}</p> : '';
```

Comment:
i dont think we should show the user nothing. maybe just default to a generic error message here